.github/workflows/pr_ci.yml

```diff
@@ -1,27 +1,26 @@
-name: PR CI
-run-name: "CI pipeline for PR - (#${{ github.event.number }}) ${{ github.event.pull_request.title }}"
+name: Release CI
+run-name: "Continous Release"
+
 
 permissions:
-  contents: read
-  actions: read
+  contents: write
+  issues: read
   id-token: write
 
 on:
-  pull_request:
-    types:
-      - opened
-      - reopened
-      - synchronize
-      - ready_for_review
-    branches: ['main']
+  workflow_dispatch:
+  # push:
+  #   branches: main
+
+concurrency: release
 
 jobs:
-  pr_ci_wrkflw:
-    name: Run CI
-    uses: PaloAltoNetworks/terraform-modules-vmseries-ci-workflows/.github/workflows/pr_ci.yml@v1.2.0
+  release_wrkflw:
```

Comment:
Why for PR CI we are going to use workflow `release_ci.yml` ?

Reply:
my bad, a leftover from tests